app/controllers/toons_controller.rb

```diff
@@ -12,6 +13,25 @@ def create
         end
     end
 
+    def update
+        toon = Toon.find(params[:id])
+        toon.update(toon_params)
+        if toon.valid?
+            render json: toon
+        else
+            render json: toon.errors, status: 422
+        end
+    end
+
+    def destroy
+        toon = Toon.find(params[:id])
+        if toon.valid?
+            toon.delete()
+            render json: toon
```

Comment:
Since the goal is to destroy that entry, recommend sending some different data to the render